override-THIRD-PARTY.properties

```diff
@@ -12,8 +12,12 @@ classworlds--classworlds--1.1-alpha-2=BSD Variant (Classworlds License)
 com.aspose--aspose-cells--21.12=Aspose EULA
 # https://about.aspose.com/legal
 com.aspose--aspose-cells--21.9=Aspose EULA
+# http://migbase64.sourceforge.net/
+com.brsanthu--migbase64--2.2=BSD-3-Clause
```

Comment:
I don't know if this is relevant, but this says BSD-2: https://github.com/brsanthu/migbase64/blob/master/LICENSE

The 2.2 tag just says "BSD" though.

Reply:
They added the 3rd clause so it became BSD-3, and I've checked on SPDX every word in the different BSD-3 licenses to detect the exact one. In fact, the 3rd clause is in the form of [BSD-3-Clause-Clear](https://spdx.org/licenses/BSD-3-Clause-Clear.html) license, while the real licensing part (in uppercase) belongs to [BSD-3-Clause](https://spdx.org/licenses/BSD-3-Clause.html). 
Hence, it's a hybrid in the form but a standard BSD-3 "new" in the substance.